src/main/java/mate/controller/DeleteDriverController.java

```diff
@@ -0,0 +1,25 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.dao.DriverDao;
+import mate.lib.Injector;
+import mate.service.DriverService;
+
+public class DeleteDriverController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private final DriverDao driverDao = (DriverDao) injector.getInstance(DriverDao.class);
+    private final DriverService driverService =
+            (DriverService) injector.getInstance(DriverService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
```

Comment:
Please rework method to work via POST method, get should be used for obtaining information, not modification